.github/renovate.json

```diff
@@ -6,7 +6,10 @@
   ],
   "regexManagers": [
     {
-      "fileMatch":  [".*\\.Dockerfile$"],
+      "fileMatch":  [
+        ".*\\.Dockerfile$",
+        ".*doc/packaging\\.md$"
```

Comment:
For what it's worth, if the regexp's aren't anchored (and the trailing `$` implies they are not, but I don't know), then the leading `.*` is useless.